### ISSUE TITLE
why your loss is very low

### ISSUE THREAD

**Issue by LPaKing** [leeyoshinari/YOLO_v2#5]

**Comment by leeyoshinari** [comment]
Because I use reduce_mean, change `loss = tf.reshape(loss, [self.batch_size, self.cell_size * self.cell_size * self.box_per_cell * (self.num_class + 5)])` and `loss = 0.5 * tf.reduce_mean(tf.reduce_mean(loss, axis = 1))` to `loss = tf.reduce_mean(tf.reduce_sum(loss, axis = [1, 2, 3, 4]), name = 'loss')`, the loss is a little bigger.

**Comment by shaoweistar** [comment]
what is the differents between this two loss

**Comment by leeyoshinari** [comment]
I think there is no difference between them.
